app/assets/javascripts/application.js

```diff
@@ -38,6 +38,7 @@
 //= require ./angular-libs/guides/directives/guides.formChecker.directive.js
 
 //= require ./angular-libs/ngs3upload
+//= require ./angular-libs/angular-socialshare
```

Comment:
The library on rails assets (https://github.com/djds4rce/angular-socialshare) isn't actually the one I'm using (https://github.com/720kb/angular-socialshare). The identical naming isn't helpful!